eng/versioning/verify_release_set.ps1

```diff
@@ -0,0 +1,97 @@
+param(
+    [Parameter(Mandatory=$true)]
+    [System.String] $ServiceDirectory,
+    # ArtifactsList will be using ('${{ convertToJson(parameters.Artifacts) }}' | ConvertFrom-Json | Select-Object name, groupId)
+    [Parameter(Mandatory=$true)]
+    [array] $ArtifactsList
+)
+
+$resultsTime = [diagnostics.stopwatch]::StartNew()
+
+# Given the service directory and a list of libraries to be released verify that the list
+# of libraries has full transitive closure. This means that we have to look at each library's
+# pom file's current dependencies and ensure that each dependency is contained within the
+# list to be released.
+$script:FoundError = $false
+$libraryReleaseList = @()
+$missingLibraries = @()
+function Write-Error-With-Color([string]$msg)
+{
+    Write-Host "$($msg)" -ForegroundColor Red
```

Comment:
Color won't show in the devops logs but if you want this to be an error and standout we should output as an error. We have some common logging functions at https://github.com/Azure/azure-sdk-for-java/blob/main/eng/common/scripts/logging.ps1 that will toggle correctly when output in DevOps or not.

Reply:
Yup, moved to LogError with the next commit.